internal/pkg/service/stream/storage/level/local/model/file.go

```diff
@@ -13,6 +12,4 @@ type File struct {
 	Assignment assignment.Assignment `json:"assignment"`
 	// Allocation configures pre-allocation of the filesystem free space.
 	Allocation diskalloc.Config `json:"allocation"`
-	// Encoding defines how is the  result format encoded, for example tabular data to CSV.
-	Encoding encoding.Config `json:"encoding"`
```

Comment:
Sorry, one more smaller refactoring, I need to pass only required configuration to the disk writer node, 

so `{file/slice}.LocalStorage.Encoding` has been moved to `{file/slice}.Encoding`, it is also related to some previous comment, in a previous PR.

Encoding is handled by the source node, in the "local" environment/level, ... but disk writer is providing "local storage",

Reply:
It's ok but now it makes more sense :smile: